export const handleError = (fullError, message) => {
    console.error(fullError, message)
}